azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+trigger:
+  - main
```

Comment:
The target branch for this PR is `master`. Are you going to rename the branch to `main`?

Reply:
Decided to keep master to avoid potential mismatch since it's a fork repo

Reply:
Understood. Btw, `main` or any other branch can always be set to track changes from the upstream repo's `master`

Reply:
Thanks for the info